CONTRIBUTING.md

```diff
@@ -1,19 +1,47 @@
-# Don't use the issue tracker to ask questions
```

Comment:
kind of an important point and the only thing showing hen you make a new issue ... so that should stay there ?